persist/sqldb/sqldb.go

```diff
@@ -50,7 +50,7 @@ func CreatePostGresDBSession(kubectlConfig kubernetes.Interface, namespace strin
 	var settings = postgresql.ConnectionURL{
 		User:     string(userNameByte),
 		Password: string(passwordByte),
-		Host:     cfg.Host + ":" + cfg.Port,
```

Comment:
@alexec I am not sure it is documented but a subtle gotcha here for users whom did not explicitly define the PG port in their config (like us) is that instead of defaulting to 5432 which I assume `host:""` would have, it now will default to port 0 as `cfg.Port` changed to an integer so it will try connect to `host:0` if you do not specify any port.

Not sure if it should be documented as a breaking change in the upgrade notes for the 2.12.0 release.

Reply:
I think it is a bug and should be fixed. Thank you for testing the RC build? Any chance you could please raise a bug?

Reply:
Sure thing can do.